roman.yaml

```diff
@@ -16,7 +16,7 @@ logging:
 swagger:
   resourcePackage: com.wire.bots.roman.resources
   schemes:
-#    - http
+    - http
     - https
```

Comment:
https should be first so it get selected by default

Reply:
and this is order dependend? I'll try that

Reply:
it is, fixed